src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/DetachedForgotPasswordPage.java

```diff
@@ -0,0 +1,49 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.DetachedWindow;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+
+public class DetachedForgotPasswordPage extends DetachedWindow implements ForgotPasswordPage {
+
+  private ForgotPasswordPage forgotPasswordPage;
+
+  private String title = ForgotPasswordPage.pageTitle;
+
+  public DetachedForgotPasswordPage(ForgotPasswordPage forgotPasswordPage) {
+    forgotPasswordPage = forgotPasswordPage;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "forgotPasswordPage". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)